dags/openshift_nightlies/scripts/run_benchmark.sh

```diff
@@ -75,8 +75,11 @@ EOF
     fi
 }
 
+# replaces '_' with '-', openshift doent support this char
+export AIRFLOW_CTX_TASK_ID=$(echo $AIRFLOW_CTX_TASK_ID | sed -r 's/[_]+/-/g')
```

Comment:
where is this var being used?

Reply:
Needed for L91(name of the benchmark workload). 
Hypershift tasks name will have `_` ex. `hosted-2_node-density-heavy` & `hosted-1_node-density-heavy`, openshift don't allow this char so replacing that with `-`

```
error: failed to create configmap: ConfigMap "kube-burner-cfg-215baf7b-hosted-2_node-density-20220210" is invalid: metadata.name: Invalid value: "kube-burner-cfg-215baf7b-hosted-2_node-density-20220210": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')
```

Reply:
can you just change the task ids then in the actual Airflow task definition instead of changing it here? We use hyphens for most of the other task names afaik

Reply:
and if we don't we should.